python/cuml/dask/DGX.md

```diff
@@ -0,0 +1,112 @@
+# Using IB for Dask + cuML MG on a DGX
+
+## 1. Install UCX from source
+
+```
+./autogen
+mkdir build && cd build
+../configure --prefix=$CONDA_PREFIX --with-cuda=/usr/local/cuda --enable-mt --disable-cma CPPFLAGS="-I//usr/local/cuda/include"
```

Comment:
Based on https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support

we may need to install https://github.com/NVIDIA/gdrcopy first

and need to provide --with-gdrcopy=<gdr_copy/install/path> in addition to --with-cuda=/usr/local/cuda

I haven't really verified this is working as expected, so take this with a grain of salt, but based on just reading, this may be necessary for faster GPU-Host and GPU-Network Card data transfer.

Reply:
And should not forget adding <gdr_copy/install/path>/lib64 to LD_LIBRARY_PATH. ucx_info -d should list gdr_copy if properly installed.

Reply:
And gdr_copy should be added to the UCX_TLS environment variable as well.

Reply:
I have added this to the markdown as well.

Reply:
Mind taking a look and sharing your thoughts?

Reply:
I added comments to the wiki/mnmg/Using_Infiniband_for_MNMG.md  file.

Reply:
> Based on https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support

Finally got a chance to look at this. This is extremely useful and I'm going to include it in the markdown.